sys-devel/gcc/gcc-5.4.0_2016_07_08.recipe

```diff
@@ -0,0 +1,347 @@
+SUMMARY="C/C++ compiler for target ${effectiveTargetMachineTriple}"
+DESCRIPTION="The standard compiler for non-legacy Haiku (i.e. for all \
+architectures other than x86_gcc2)."
+HOMEPAGE="http://gcc.gnu.org"
+
+SOURCE_URI="https://ftp.gnu.org/gnu/gcc/gcc-5.4.0/gcc-5.4.0.tar.bz2"
```

Comment:
Well, since we have CHECKSUM_SHA256 we don't really need https for SOURCE_URI.
Of course, if the mirror rediretor was available in https I would use it, but it is not.
IMHO using the mirror redirector, even on http, is much more interesting because it both speeds up the download time and makes it more resilient in case the main GNU site becomes unavailable.
The only real (but small) problem with the mirror redirector is that in very rare cases the (random) mirror that we could be redirected to could be missing the archive. That happened to me twice in almost 6 months. And the workaround is easy: relaunch haikuporter ;-)